packages/integration-karma/test/light-dom/slotting/index.spec.js

```diff
@@ -53,3 +54,14 @@ describe('imperative slotted content', () => {
         });
     });
 });
+
+describe('in shadow dom', () => {
```

Comment:
While this regression was introduced because of the restriction related to light DOM, this test has nothing to do with light DOM.